docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md

```diff
@@ -188,34 +200,29 @@ The `request_ip` field is optional.
 
 ## Step 4: Embed Checkout
 
-Use the `embedded_checkout_url` that is returned and assemble a JSON object that will be used by the Checkout JS SDK to determine how to render the checkout. Pass the object to the `embedCheckout` method of the Checkout SDK. This will render the checkout to an HTML element with the `id` you chose.
+Use the `embedded_checkout_url` that is returned from generating redirect URLs and assemble a JSON object. It will be used by the Checkout JS SDK to determine how to render the checkout. Pass the object to the `embedCheckout` method of the Checkout SDK. This will render the checkout to an HTML element with the `id` you chose.
```

Comment:
```suggestion
Assemble a JSON object using the `embedded_checkout_url` returned from generating redirect URLs. It will be used by the Checkout JS SDK to determine how to render the checkout. 

<Corresponding code sample should go here.>

Pass the object to the `embedCheckout` method of the Checkout SDK. This will render the checkout to an HTML element with the `id` you chose.

<Code sample here.>
```